invoices/invoiceregistry.go

```diff
@@ -95,21 +99,51 @@ func NewRegistry(cdb *channeldb.DB, finalCltvRejectDelta int32) *InvoiceRegistry
 		hodlSubscriptions:         make(map[channeldb.CircuitKey]map[chan<- interface{}]struct{}),
 		hodlReverseSubscriptions:  make(map[chan<- interface{}]map[channeldb.CircuitKey]struct{}),
 		finalCltvRejectDelta:      finalCltvRejectDelta,
+		expiryWatcher:             expiryWatcher,
 		quit:                      make(chan struct{}),
 	}
 }
 
+// prefetchPending fetches all active invoices and their corresponding payment
+// hashes from ChannelDB and adds them to the expiry watcher.
+func (i *InvoiceRegistry) prefetchPendingInvoices() error {
```

Comment:
Function name doesn't fully cover what it does. It doesn't just fetch, but also primes the watcher.

Reply:
Good point, renamed it to better express what it does.